packages/shared_preferences/shared_preferences_linux/pubspec.yaml

```diff
@@ -11,17 +11,17 @@ flutter:
         pluginClass: none
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-259.8.beta <3.0.0"
   flutter: ">=1.12.8"
 
 dependencies:
-  file: ">=5.1.0 <7.0.0"
   flutter:
     sdk: flutter
+  file: ">=6.0.0-nullsafety.4 <7.0.0"
```

Comment:
This can be written with the usual ^ notation. The previous version was to span two different major versions.

Reply:
Noted